docs/ruleset_functions/get_zone_supply_return_exhaust_relief_terminal_fan_power_dict.md

```diff
@@ -0,0 +1,87 @@
+# get_zone_supply_return_exhaust_relief_terminal_fan_power_dict
+
+**Description:** Get the supply, return, exhaust, relief, and terminal total fan power for each zone. The function returns a dictionary that saves each zone's supply, return, exhaust, relief  and terminal unit fan power as a list {zone.id: [supply fan power kW, return fan power kW, exhaust fan power kW, relief fan power kW, terminal fan power]}. Values will be equal to zero where not defined for a fan system.
```

Comment:
Should we include zonal exhaust fan and non-mechanical cooling fan types to this function also? Now these are handled directly by the RDS.

Reply:
Maybe change after multiple fan discussion.

Reply:
Done